newscoop/themes/system_templates/editor_image.tpl

```diff
@@ -1,6 +1,6 @@
 {{ dynamic }}
 	{{ if $imageDetails['align'] }} <div align="center"> {{ /if }}
-		<div class="cs_img {{ if $imageDetails['align'] }}cs_fl_{{ $imageDetails['align'] }}{{ /if }}" style="{{ if $imageDetails['width'] }}width:{{ $imageDetails['width'] }};{{ /if }}">
+		<div class="cs_img {{ if $imageDetails['align'] }}cs_fl_{{ $imageDetails['align'] }}{{ /if }}" {{ if $imageDetails['percentage'] }}style="width:{{ $imageDetails['percentage'] }};"{{ /if }}>
```

Comment:
yeah, but sometimes you need that width in px. For example in image centering when it behaves as block. You need it also for initialing image size before it will be fetched and rendered.